### ISSUE TITLE
QS overall lnegth limit not documented

### ISSUE THREAD

**Issue by sergeig888** [MicrosoftDocs/powerbi-docs#2372]
Apparently overall length of the URL that includes QS filters can't exceed 2048 characters.  This needs to be confirmed with the engineering team PM owner and added to this doc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 70b360f3-21a7-adc2-896a-b1dfd5d9e384
* Version Independent ID: 15e8c09f-46ad-68f9-75e6-c3d5c2c214e5
* Content: [Filter a report using query string parameters in the URL - Power BI](https://docs.microsoft.com/en-us/power-bi/collaborate-share/service-url-filters)
* Content Source: [powerbi-docs/collaborate-share/service-url-filters.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/collaborate-share/service-url-filters.md)
* Service: **powerbi**
* Sub-service: **powerbi-service**
* GitHub Login: @maggiesMSFT
* Microsoft Alias: **maggies**

**Comment by davidiseminger** [comment]
Thanks for your comment, @sergeig888. We’ve forwarded your feedback to the writer, who should respond to this thread shortly. Thanks!

**Comment by maggiesMSFT** [comment]
Hi, @sergeig888 -- checked with the product team. Turns out max URL length is a browser limitation, not a Power BI limitation. 
Internet Explorer: 2,048 chars
Edge: 81,578
Firefox: 65,536 and counting
Safari: 80,000 and counting
Opera: 190,000 and counting
Apache: 4,000
Chrome: stops *displaying* after 64,000 but can serve >100,000

This Stack Overflow article has the numbers: https://stackoverflow.com/questions/812925/what-is-the-maximum-possible-length-of-a-query-string

I added a note to the article. 

Thanks! and I hope that helps!
